src/main/java/ru/sber/entity/Category.java

```diff
@@ -0,0 +1,31 @@
+package ru.sber.entity;
+
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.*;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "Categories",
+        uniqueConstraints = {
+                @UniqueConstraint(columnNames = "name")
+        })
+public class Category {
+    @Id
+    @Nonnull
```

Comment:
Эти аннотации здесь не работают
Если ты хотела указать, что поле в таблице не может быть null, то нужно написать 
```suggestion
    @Column(nullable = false)
```
Если хочешь, чтобы была проверка поля в самой java, то используй spring-boot-starter-validation
https://www.baeldung.com/spring-boot-bean-validation

```suggestion
    @NotNull(message = "Идентификатор не может быть null")
```

Reply:
исправила